src/store/getters.ts

```diff
@@ -337,7 +337,7 @@ export const getConversionTypeOptions = (state: StateIF): ConversionTypesI[] =>
     options = options.concat(getConversionTypeAddToSelect(state))
     n = 4
   }
-  options = options.concat({ text: 'View all Alterations', value: 'INFO', rank: n })
+  options = options.concat({ text: 'View all Alterations', value: 'INFO' as any, rank: n })
```

Comment:
The type of `value`, an enum, doesn't have the INFO value, so we can do this.

It's better to declare an enum value for INFO as in EntityType enum, but I'm not going to re-write all of Namerequest UI 😛